src/runtime_src/core/pcie/tools/xbmgmt/cmd_flash.cpp

```diff
@@ -690,17 +694,44 @@ static int shell(int argc, char *argv[])
             file = std::string(optarg);
             break;
```

Comment:
don't you want to remove these lines so make 1 and 2 the same? thus you have one less case to worry about later when you validate the arguments...

Reply:
True, but we cannot find out the issues if both --path/--primary is given, but thats a corner case which we can ignore at this point of time as we have removed that --path from help also.. I will make changes.